api_v3/lib/KalturaLiveEntryService.php

```diff
@@ -341,7 +341,12 @@ private function createRecordedEntry(LiveEntry $dbEntry, $mediaServerIndex)
 			$recordedEntry->setModerationStatus($dbEntry->getModerationStatus());
 			$recordedEntry->setIsRecordedEntry(true);
 			$recordedEntry->setTags($dbEntry->getTags());
-			
+
+			// make the recorded entry to be "hidden" in search so it won't return in entry list action
+			if ($dbEntry->getRecordingOptions() && $dbEntry->getRecordingOptions()->getShouldMakeHidden())
```

Comment:
this is specific to live entries, so doing it somewhere else will generate more code lines for checks and will be less object oriented since I'll have to check types